coralogix-policies/coralogix-remote-actions-policy/template.yaml

```diff
@@ -0,0 +1,104 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: The module will create a role with an inline policy to allow Coralogix to execute custom actions remotely.
+Parameters:
+  AwsAccountId:
+    Description: "Aws Account id"
+    Type: "String"
+  ExternalId:
+    Description: "ExternalId for sts:AssumeRole"
+    Type: "String"
+  CoralogixRegion:
+    Type: String
+    Default: EU1
+    Description: The AWS account that you want to deploy the integration in.
+    AllowedValues:
+      - dev
```

Comment:
We usally dont put dev/staging and use customeendpoint for such cases

Reply:
I just copied what's in the other aws metrics related template - but I can delete it from both if necessary.

Reply:
removed ✅